json_rpc/private/jrpc_conv.nim

```diff
@@ -0,0 +1,94 @@
+# json-rpc
+# Copyright (c) 2023 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+import
+  std/[tables, macros],
+  results,
+  json_serialization,
+  json_serialization/stew/results as jser_results,
+  json_serialization/std/[options, sets, tables]
+
+export
+  options,
+  sets,
+  tables,
+  json_serialization,
+  jser_results
+
+type
+  StringOfJson* = JsonString
+
+createJsonFlavor JrpcConv,
+  automaticObjectSerialization = true,
```

Comment:
in nimbus we've had several cases of objects not intended for serialization end up getting caught in these generic open-for-all serializers - this is an edge case because _technically_ this serializer looks like it's only used during parameter conversion which in a way is an explicit choice - nonetheless, having it accept any object seems risky.

Reply:
fixed